kex-runner/src/main/kotlin/org/vorpal/research/kex/jacoco/minimization/GreedyAlgorithm.kt

```diff
@@ -0,0 +1,64 @@
+package org.vorpal.research.kex.jacoco.minimization
+
+import java.nio.file.Path
+
+interface TestReduction
+
+class Test(
```

Comment:
I would either rename `Test` and `Requirements` classes to be more descriptive (e.g. `TestCoverageInfo`), or made them `private` so that they does not cause confusion in other parts of the code.